### PR TITLE
Added reference to snapshot services

### PR DESCRIPTION
**Pull request by stakeworld**

### CODE REVIEW

File: docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md [hunk 1]
```diff
@@ -411,6 +411,27 @@ verified. You can then compare that to the current highest block via
 [Telemetry](https://telemetry.polkadot.io/#list/Kusama) or the
 [Polkadot-JS Block Explorer](https://polkadot.js.org/apps/#/explorer).
 
+#### Database Snapshot Services
+
+If you start a node for the first time, it will start building from the genesis block. This process can 
+take a while depending on the database size. To make this process faster, snapshots can be used. Snaphots
```
Comment (joepetrowski) [inline docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md]:
```suggestion
take a while depending on the database size. To make this process faster, snapshots can be used. Snapshots
```

File: docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md [hunk 2]
```diff
@@ -411,6 +411,27 @@ verified. You can then compare that to the current highest block via
...
+are compressed backups of the database directory of Polkadot/Kusama nodes, containing either the whole chain 
+(or a pruned version of it, with only the latest 1000 or 256 blocks). Listed below are a few public snapshot
```
Comment (joepetrowski) [inline docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md]: They still have all the blocks, it's just the last <pruned> _states_.

File: docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md [hunk 3]
```diff
@@ -411,6 +411,27 @@ verified. You can then compare that to the current highest block via
...
+
+- [STAKEWORLD](https://stakeworld.nl/docs/snapshot)
+- [Polkachu](https://polkachu.com/snapshots)
+- [Polkashots](https://polkashots.io/)
+
+:::caution 
+
+For the security of the network, it is recommended that you sync from scratch, even if you are running your 
+node in pruning mode for validation. The reason is that if these snapshots get corrupted and a majority of nodes 
+run based on these screenshots, the network could end up running on a non-canonical chain.
```
Comment (joepetrowski) [inline docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md]:
```suggestion
run based on these snapshots, the network could end up running on a non-canonical chain.
```

**Review by DrW3RK — APPROVED** [review]
Thank you @stakeworld

**Pull request merged**